### PR TITLE
Update iOS 17+ support on COLR/CPAL v0

### PR DESCRIPTION
**Pull request by rutopio**
Reference: COLR support vanished between Safari 16 and 17: [WebKit Bugzilla Bug 262223](https://bugs.webkit.org/show_bug.cgi?id=262223)

### CODE REVIEW

**Comment by Schweinepriester** [comment]
Fixes https://github.com/Fyrd/caniuse/issues/6878.

File: features-json/colr.json [hunk 1]
```diff
@@ -508,9 +508,9 @@
       "16.4":"y",
       "16.5":"y",
       "16.6":"y",
-      "17.0":"y",
-      "17.1":"y",
-      "17.2":"y"
+      "17.0":"n #4",
+      "17.1":"n",
+      "17.2":"n"
```
Comment (Schweinepriester) [inline features-json/colr.json:513]: I think it makes sense to put the note on all versions starting 17, assuming it's not intended to be removed.
If it turns out to be intended I guess just `n` it is.
But for now:

```suggestion
      "17.0":"n #4",
      "17.1":"n #4",
      "17.2":"n #4"
```

File: features-json/colr.json [hunk 2]
```diff
@@ -590,6 +590,7 @@
     "1":"Supported only on Safari for macOS High Sierra (10.13)+",
     "2":"Supported only on Windows 8.1+",
     "3":"On Windows pre 8.1 and macOS Sierra (10.12) and below falls back to using FreeType: [Chromium bug 882844](https://bugs.chromium.org/p/chromium/issues/detail?id=882844#c3)"
+    "4":"COLR support vanished between Safari 16 and 17: [WebKit Bugzilla Bug 262223](https://bugs.webkit.org/show_bug.cgi?id=262223)"
```
Comment (Schweinepriester) [inline features-json/colr.json:593]:
```suggestion
    "4":"COLRv0 support vanished with Safari 17: [WebKit Bugzilla Bug 262223](https://bugs.webkit.org/show_bug.cgi?id=262223)"
```

**Review by Schweinepriester — CHANGES_REQUESTED** [review]
I have some suggestions - an additional one for which I can't create a code suggestion is to also change (desktop) Safari to `n #4` starting with 17.0 all the way to TP, please.

https://github.com/Fyrd/caniuse/issues/1018#issuecomment-1737111439 appears to show desktop (as well).

**Comment by rutopio** [comment]
I am certain that iOS 17 does not support on my device. 
Currently, I am running macOS Monterey, and COLRv0 works perfectly on it. 
Need additional feedback from users with the latest macOS versions.

**Comment by Schweinepriester** [comment]
Even with Safari 17!?
Funky, okay, then we might need a distinction.

Perhaps we can summon @t3chguy: On which version of macOS did you take the second screenshot in https://github.com/Fyrd/caniuse/issues/1018#issuecomment-1737111439? :)

**Comment by rutopio** [comment]
Sorry for the confusion 😯, I meant that macOS Monterey with Safari 16, and it works perfectly. However, iOS 17 with Safari 17 fail.
That's very possible that both Desktop and Mobile Safari 17 do not support due to the latest Webkit kernel.
Perhaps @t3chguy can provide more details about the desktop side.

**Comment by t3chguy** [comment]
image

Here you go

**Comment by rutopio** [comment]
Thanks!
I've marked both Desktop and Mobile Safari versions above 17 as unsupported.

**Comment by Fyrd** [comment]
Thanks!

File: features-json/colr.json [hunk 3]
```diff
@@ -590,6 +590,7 @@
     "1":"Supported only on Safari for macOS High Sierra (10.13)+",
     "2":"Supported only on Windows 8.1+",
     "3":"On Windows pre 8.1 and macOS Sierra (10.12) and below falls back to using FreeType: [Chromium bug 882844](https://bugs.chromium.org/p/chromium/issues/detail?id=882844#c3)"
```
Comment (Fyrd) [inline features-json/colr.json:592]:
```suggestion
    "3":"On Windows pre 8.1 and macOS Sierra (10.12) and below falls back to using FreeType: [Chromium bug 882844](https://bugs.chromium.org/p/chromium/issues/detail?id=882844#c3)",
```

**Pull request merged**